docs/DeveloperGuide.md

```diff
@@ -532,133 +535,183 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+    * 1a2. User makes another request to unlink a friend with a game, with a different `FRIEND_ID`/`GAME_ID`.  
+    * Use case continues from step 2.
 
+* 1b. gitGud detects that the `FRIEND_ID` and`GAME_ID` link does not exist.
+    * 1b1. gitGud informs user that it has no friend and game link was found with specified `FRIEND_ID` and `GAME_ID`.  
+    * 1b2. User makes another request to unlink a friend with a game, with a different `FRIEND_ID` and/or `GAME_ID`.  
+    * Use case continues from step 2.
 
-**Use case: UC04 - List games whose `GAME_ID` contains a keyword**
+**Use case: UC06 - Assign a skill value for a linked game to a friend**
 
 **MSS**
 
-1. User requests to list games in gitGud using a keyword filter.
-2. gitGud shows a filtered list of games whose `GAME_ID` contains the keyword.
+1. User requests to assign a `SKILL_VALUE` to a friend from a linked game using a friend's `FRIEND_ID`, a game's `GAME_ID`.
+2. gitGud assigns the friend a skill value for the linked game and informs user of successful assigning of skill value.
 
    Use case ends.
 
 **Extensions**
 
-* 2a. The user does not provide a keyword.
+* 1a. gitGud detects that the `FRIEND_ID`/`GAME_ID` does not exist.
+    * 1a1. gitGud informs user that it has no friend/game was found with specified `FRIEND_ID`/`GAME_ID`.  
+    * 1a2. User makes another request to assign a skill value, with a different `FRIEND_ID`/`GAME_ID`.  
+    * Use case continues from step 2.
 
-    * 2a1. All games are listed.
+* 1b. gitGud detects that the `FRIEND_ID` and`GAME_ID` link does not exist.
+    * 1b1. gitGud informs user that it has no friend and game link was found with specified `FRIEND_ID` and `GAME_ID`.  
+    * 1b2. User makes another request to assign a skill value a friend with a game, with a different `FRIEND_ID` and/or `GAME_ID`.  
+    * Use case continues from step 2.
 
-      Use case ends.
-
-
-**Use case: UC05 - Link a friend to a particular game**
+**Use case: UC07 - Schedule a friend**
 
 **MSS**
 
-1. User links a friend (using `FRIEND_ID`) with a particular game (using `GAME_ID`) and the username for that game 
-   (using `USERNAME`).
-2. gitGud associates the friend with the game provided.
+1. User requests to update a friend's schedule using a friend's `FRIEND_ID`, the time period (`START_HOUR` to `END_HOUR`) of the `DAY` they wish to set, as well as whether the friend `IS_FREE` then.
+2. gitGud updates the friend's schedule and informs user of successful scheduling of friend.
 
-    Use case ends.
+   Use case ends.
 
 **Extensions**
 
-* 2a. The given `FRIEND_ID` is invalid.
+* 1a. gitGud detects that the `FRIEND_ID` does not exist.
+    * 1a1. gitGud informs user that it has no friend was found with specified `FRIEND_ID`.  
+    * 1a2. User makes another request to schedule a friend with a different `FRIEND_ID`.  
+    * Use case continues from step 2.
 
```

Comment:
I think this is missing the invalid input for the periods as well as the free number.